include/cantera/base/Storage.h

```diff
@@ -58,6 +58,10 @@ class Storage
     //! @param permissive  if true, do not raise exceptions
     bool checkGroup(const string& id, bool permissive=false);
 
+    //! Unlink group
+    //! @param id  storage location within file
+    void unlinkGroup(const string& id);
```

Comment:
I'm guessing that "unlink" is jargon for "delete"? I think the latter would be a better name -- even the base HDF5 API seems to be moving in this direction (see [here](https://docs.hdfgroup.org/hdf5/v1_12/group___h5_g.html#gacb843cbd5bbb816cfa9c855463d1e51c)).

Reply:
Python uses `unlink` also, but I’m not opposed to changing this whatsoever.